torch/_meta_registrations.py

```diff
@@ -2698,6 +2698,9 @@ def activate_meta():
                 _meta_lib_dont_use_me_use_register_meta_for_mkldnn.impl(op_overload, fn)
             elif "mkl::" in op_overload.name():
                 _meta_lib_dont_use_me_use_register_meta_for_mkl.impl(op_overload, fn)
+            elif "quantized_decomposed::" in op_overload.name():
+                lib = torch.library.Library("quantized_decomposed", "IMPL", "Meta")
```

Comment:
nit, maybe rewrite this to only load lib once... 
I don't know how expensive it is to create a lib every iteration.